test/lib/foreman/logging_test.rb

```diff
@@ -7,6 +7,17 @@ def test_configure_once
     end
   end
 
+  def test_environment_not_found_yields_defaults
+    assert_nothing_raised do
+      Foreman::Logging.instance_variable_set("@configured", false)
+      Foreman::Logging.configure(:environment => 'foo')
```

Comment:
This has the small side-effect of actually creating a log/foo.log, can we prevent that?